hvplot/tests/testplotting.py

```diff
@@ -50,3 +54,28 @@ def test_pandas_dataframe_plot_does_not_implement_pie(self):
 class TestPandasHvplotPlotting(TestPandasHoloviewsPlotting):
     def setUp(self):
         pd.options.plotting.backend = 'hvplot'
+
+
+def test_plot_supports_polars():
```

Comment:
Why did you add this? Maybe you didn't know about 

https://github.com/holoviz/hvplot/blob/266565b9b54414b7eed62203dcfcdc460f098505/hvplot/tests/plotting/testcore.py#L84

Reply:
I guess I was confused by the merge conflict.